client/src/components/profileInfoEdit/ProfileClearModal.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Delete Alert Modal component display a modal and ask for confirmation when the delete button is clicked
+ */
+
+import React from 'react';
+import toast from 'react-hot-toast';
+import { Button, Modal } from 'react-bootstrap';
+import { useSelector, useDispatch } from 'react-redux';
+import { clearTeam, deleteGHPages } from '../../actions/team';
+
+const ProfileClearModal = ({ clearAlert, setclearAlert }) => {
+  const { teamId } = useSelector((state) => state.team);
+  const dispatch = useDispatch();
+
+  const HandleClear = () => {
+    const access_token = localStorage.getItem('GH_access_token');
```

Comment:
Try to avoid camel case here, I know it's suppressed in the eslint rules but its pending refactor so if we use snake case here we can avoid more work for that eventual refactor. :)

Reply:
which variables are you reffereing to tho? ihve had the clearAlert etc in camel case to follow the consistency but yeah can you point out the variable you are trying to refer to?

Reply:
Sorry, I meant avoid snake case LOL (don't know what I was thinking when I wrote that)
`access_token` should be `accessToken`

Reply:
Right xp